src/knockout/kosurvey.ts

```diff
@@ -90,9 +88,9 @@ export class SurveyImplementor extends ImplementorBase {
       this.renderedElement
     );
   }
-  public koEventAfterRender(element: any, survey: any) {
+  public koEventAfterRender(element: any) {
     SvgRegistry.renderIcons();
-    survey.afterRenderSurvey(element);
+    this.survey.afterRenderSurvey(element);
```

Comment:
Probably this is wrong here. We need to use second argument: `survey: SurveyModel`

Reply:
The reason that it doesn't work is because of the way knockoutJS calls functions on a class. The framework predates actual classes and takes the method and calls it without binding it.

The cleaner solution imo is to fix this in the template:
```
<!-- ko template: { name: "survey-content-template", afterRender: $data.implementor.koEventAfterRender } -->

<!-- ko template: { name: "survey-content-template", afterRender: function() { $data.implementor.koEventAfterRender() } } -->
```

The second approach masks this ugliness and keeps the class clean. (For testing this also means that you could test if trigger the afterRender event calls the expected callbacks in the `SurveyModel` and `SvgRegistry`)